pandas/core/series.py

```diff
@@ -2138,6 +2138,14 @@ def map(self, arg, na_action=None):
         two   bar
         three baz
 
+        >>> z
```

Comment:
Can you add a single sentence between the examples? Like "Mapping a dictionary works similar as with a Series (mapping dict keys on index labels): "